docs/src/modules/components/TopLayoutCareers.js

```diff
@@ -42,10 +51,11 @@ export default function TopLayoutCareers(props) {
             href="/careers/#open-roles"
             rel="nofollow"
             variant="body2"
-            sx={{ display: 'block', mb: 2 }}
+            sx={{ display: 'flex', gap: 0.5, alignItems: 'center', mb: 2 }}
           >
+            <KeyboardArrowLeftIcon fontSize="small" />
             {/* eslint-disable-next-line material-ui/no-hardcoded-labels */}
```

Comment:
We could remove this `eslint-disable` by moving "Back to open roles" to the translations file